packages/lisk-transactions/src/utils/validation/validate_transaction.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { ErrorObject, ValidateFunction } from 'ajv';
+import { PartialTransaction } from '../../types/transactions';
+import * as schemas from './schema';
+import { validator } from './validator';
+
+interface BaseTransaction {
+	readonly type: number;
+}
+
+interface MultiSignatureTransaction extends BaseTransaction {
```

Comment:
I think you've already defined this interface in type definition file for transactions